receiver/hostmetricsreceiver/internal/scraper/processscraper/process_constants.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processscraper
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+// process metric constants
+
+const (
+	cmdlineLabelName   = "command_line"
+	directionLabelName = "direction"
+	processLabelName   = "process"
+	stateLabelName     = "state"
+	usernameLabelName  = "username"
+)
+
+const (
+	readDirectionLabelValue  = "read"
+	writeDirectionLabelValue = "write"
+)
+
+const (
+	userStateLabelValue   = "user"
+	systemStateLabelValue = "system"
+	waitStateLabelValue   = "wait"
+)
+
+var metricCPUUsageDescriptor = createMetricCPUUsageDescriptor()
+
+func createMetricCPUUsageDescriptor() pdata.MetricDescriptor {
```

Comment:
These are not constants. Maybe just utils.go? Not sure what is the right name.

Reply:
Yea good point - maybe `metadata.go`? _matching the proposal https://github.com/open-telemetry/opentelemetry-collector/issues/985#issuecomment-632307672_

I'll leave this refactoring to another PR though as its the same across all the scrapers.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector/pull/1074